lib/fetch/index.js

```diff
@@ -2098,12 +2099,21 @@ async function httpNetworkFetch (
           // TODO (fix): Do we need connection here?
           const { connection } = fetchParams.controller
 
+          // TODO: pass connection timing info
```

Comment:
Is there something missed to add this?

Reply:
According to [2.6](https://fetch.spec.whatwg.org/#connections) in the spec, each connection should have an associated timing info, with values for "domain lookup start time", "domain lookup end time", "connection start time" and "connection end time". I wasn't able to find any place where these values are tracked today, and I didn't dig deep enough into the connection process to understand where to add them.

Step 7.2 under [HTTP-network fetch](https://fetch.spec.whatwg.org/#http-network-fetch), says we should use the connection timing info together with a "defaultStartTime" equal to postRedirectStartTime which is used as the lower bound for each of these values. So without these values I understand that the postRedirectStartTime (which we have) is a sensible default until these values are tracked in more detail.

Reply:
Can you add those as comments, it can provide great context to understand what's left

Reply:
Added comments to this and the two other places where I add timing info